go.mod

```diff
@@ -0,0 +1,68 @@
+module github.com/stellar/kelp
+
+go 1.13
+
+require (
+	cloud.google.com/go v0.84.0 // indirect
+	github.com/Beldur/kraken-go-api-client v0.0.0-20180126083054-8d8ccfe4cc60
```

Comment:
Seems reasonable — any dependency version updated here or it’s a direct port of the previous versions?

Reply:
Direct port all the versions should be the same